charts/node-analyzer/values.yaml

```diff
@@ -92,6 +92,9 @@ nodeAnalyzer:
     - effect: NoSchedule
       key: node-role.kubernetes.io/master
 
+  # Set nodeAnalyzer daemonset priorityClassName
+  priorityClassName:
```

Comment:
no {} here?

Reply:
since this is supposed to be a string, we could set a default `""`, but it's not strictly necessary since it only gets templated if the value is [set](https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/templates/daemonset-node-analyzer.yaml#L66)

Reply:
ah